tests/test_schema.py

```diff
@@ -158,9 +164,9 @@ def test_include_data_with_single(self, post):
         assert 'first_name' in author['attributes']
 
     def test_include_data_with_all_relations(self, post):
-        data = PostSchema(include_data=('author', 'post_comments')).dump(post).data
+        data = PostSchema(include_data=('author', 'post_comments', 'post_comments.author')).dump(post).data
         assert 'included' in data
-        assert len(data['included']) == 3
+        assert len(data['included']) == 5
```

Comment:
Could you also add an assertion that the correct object is serialized here, i.e. the nested `authors`?